website/index.html

```diff
@@ -44,6 +44,16 @@ <h3 id="introduction">Introduction</h3>
   activity.footer = (android.widget.TextView) activity.findViewById(2130968579);
   activity.title = (android.widget.TextView) activity.findViewById(2130968577);
 }</pre>
+            <h4 id="resource">Resource Binding</h4>
+            <p> Bind a pre-defined resources with <code>@BindBool</code>, <code>@BindColor</code>, <code>@BindDimen</code>, <code>@BindDrawable</code>, <code>@BindInt</code>, <code>@BindString</code>, which binds an <code>R.bool</code> ID(or your specified type) to its corresponding field.</p>
```

Comment:
There's still a leading space after `<p>` and you need a space between "ID" and the opening paren `(`.